index.html

```diff
@@ -8,46 +8,82 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Daria Pavlyuk Portfolio</title>
     <link rel  = "preconnect" href ="https://fonts.googleapis.com">
+    <link rel ="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link href ="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel ="stylesheet" href = "css/index.css">
     <link rel ="stylesheet" href ="css/media.css">
+    <link rel="stylesheet" href="https://https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    
   </head>
   <body>
+    <section>
     <nav id="navbar">
       <ul>
         <li><a href="#About">About</a></li>
         <li><a href="#Experience">Experience</a></li>
         <li><a href="#Connect">Connect</a></li>
+        <li><a href="#Skills">Skills</a></li>
+        <li><a href="#Projects">Projects</a></li>
       </ul>
   </nav>
-    <div class = "wrapper">
+    <div class = "wrapper"> 
 <h1>Daria Pavlyuk</h1>
+
     <img src="img/myphoto.jpeg" alt="Daria Personal Photo" />
+
+    
+      <p><em>"Programming is the art of algorithm design and the craft of debugging errant code"- Ellen Ullman</em></p>
+       <p><em>"Everybody should learn to program a computer because it teaches you how to think"-Steve Jobs</em></p>
+      <p><em>"Programming isn't about what you know: it's about what you can figure out" - Chris Pine</em></p>
     <section id="About">
       <h2>About:</h2>
       <p class="aboutMe">
-        My name is Daria Pavlyuk. I am a QA Engineer. I <em>enjoy</em> testing
-        Web Apps. I am a student at CTD. Coding is <strong>FUN </strong>!
+        My name is Daria Pavlyuk. I am an ISTQB-certified QA Software Engineer dedicated to ensuring excellence in Software Quality for an impeccable user experience. <em>Knowing</em> that I am helping to make people's lives easier or better is a great source of motivation for me. I am a student at CTD. Coding is <strong> a new adventure in my life </strong> and I am deeply <strong>passionate</strong> about it! I love the challenge and creativity involved in solving problems and building projects from the ground up. Every day brings new opportunities to expand my knowledge and improve my skills!
       </p>
     </section>
     <section id="Experience">
       <h2>Experience:</h2>
-      <ul class="experience">
-        <li>March, 2023 - until present - Utest.</li>
-      </ul>
+      <div class="flex-container"><!--Container Experinece-->
```

Comment:
The instructions for the task require your experience to stay in rows and columns, find below, the HTML syntax for it, and also check out the CSS

 <section id="Experience">
                <h2>Experience:</h2>

                <div class="grid-container">
                    <div class="item-1">QA Tester </div>
                    <div class="item-2">TestPro</div>
                    <div class="item-3">Jan 2021 - Feb 2022</div>
                </div>

                <div class="grid-container">
                    <div class="item-1">Sotware QA Engineer</div>
                    <div class="item-2"> LIX RealEstate </div>
                    <div class="item-3">2022 - Present</div>
                </div>

                <div class="grid-container">
                    <div class="item-1">UAT Analyst</div>
                    <div class="item-2">DermDivision Office</div>
                    <div class="item-3">2015 - Present</div>
                </div>

                <div class="grid-container">
                    <div class="item-1">Student at CTD</div>
                    <div class="item-2">Learning Programming Fundamentals</div>
                    <div class="item-3">May 2024 - Present</div>
                </div>

            </section>